conf/pulsar_env.sh

```diff
@@ -60,6 +60,9 @@ for token in $("$JAVA_BIN" -version 2>&1 | grep 'version "'); do
           JAVA_MAJOR_VERSION=${BASH_REMATCH[1]}
         fi
         break
+    elif [[ $token =~ \"([[:digit:]]+)(.*)\" ]]; then
```

Comment:
It would be nice to add some comment about this case here.

Reply:
What about such comment: `# Process the java versions without dots, such as "17-internal".`